### PR TITLE
fix(deisctl) : deisctl Journal and status custom error message for global units

### PR DESCRIPTION
**Pull request by smothiki**
Fixes https://github.com/deis/deis/issues/2917

### CODE REVIEW

**Comment by mboersma** [comment]
@smothiki could you add a newline at the end of each message? `fmt.Printf` doesn't:
```console
$ deisctl journal store-daemon
Unable to get status for global unit deis-store-daemon.service. Check the logs on the host using journalctl.gradishar:deis matt$ 
```
Also could you `git commit --amend` [the subject](http://docs.deis.io/en/latest/contributing/standards/#commit-style) to be `fix(deisctl): add custom error message for global units`?

**Comment by smothiki** [comment]
Sorry , Will change the commit subject and add new line .

**Comment by mboersma** [comment]
No worries, I was just testing and reviewing. Thanks for fixing this!

File: deisctl/backend/fleet/journal.go [hunk 1]
```diff
@@ -25,6 +25,10 @@ func runJournal(name string) (exit int) {
 		fmt.Fprintf(os.Stderr, "Error retrieving Unit %s: %v", name, err)
 		return 1
 	}
+	if suToGlobal(*u) {
+		fmt.Fprintf(os.Stderr, "Unable to get status for global unit %s. Check the logs on the host using journalctl.", name)
```
Comment (carmstrong) [inline deisctl/backend/fleet/journal.go]: This should be something like "Unable to get journal for global unit" -- we replaced status in the second sentence, but not the first.

**Comment by mboersma** [comment]
These are much better error messages when dealing with global units through `deisctl`. Code LGTM.

**Comment by carmstrong** [comment]
Thanks for fixing this up, @smothiki. Code LGTM.

**Pull request merged**